test/jdk/lib/client/ExtendedRobot.java

```diff
@@ -334,62 +158,4 @@ public void dragAndDrop(int fromX, int fromY, int toX, int toY){
     public void dragAndDrop(Point from, Point to){
         dragAndDrop(from.x, from.y, to.x, to.y);
     }
-
-    /**
-     * Successively presses and releases a given key.
-     * <p>
-     * Key codes that have more than one physical key associated with them
-     * (e.g. {@code KeyEvent.VK_SHIFT} could mean either the
-     * left or right shift key) will map to the left key.
-     *
-     * @param   keycode Key to press (e.g. {@code KeyEvent.VK_A})
-     * @throws  IllegalArgumentException if {@code keycode} is not
-     *          a valid key
-     *
-     * @see     java.awt.Robot#keyPress(int)
-     * @see     java.awt.Robot#keyRelease(int)
-     * @see     java.awt.event.KeyEvent
-     */
-    public void type(int keycode) {
-        keyPress(keycode);
-        waitForIdle(DEFAULT_SPEED);
-        keyRelease(keycode);
-        waitForIdle(DEFAULT_SPEED);
-    }
-
-    /**
-     * Types given character
-     *
-     * @param   c   Character to be typed (e.g. {@code 'a'})
-     *
-     * @see     #type(int)
-     * @see     java.awt.event.KeyEvent
-     */
-    public void type(char c) {
```

Comment:
Those `type(char...` are not migrated also. 

At least one test uses it:

https://github.com/alisenchung/jdk/blob/8150564/test/jdk/java/awt/Window/ShapedAndTranslucentWindows/ShapedTranslucentWindowClick.java#L178

With your change, it now calls `type(int)` directly, which has a different implementation.